debian/cloud-init.maintscript

```diff
@@ -6,4 +6,4 @@ rm_conffile /etc/init/cloud-init-local.conf 0.7.9-243-ge74d775-0ubuntu2~
 rm_conffile /etc/init/cloud-init-nonet.conf 0.7.9-243-ge74d775-0ubuntu2~
 rm_conffile /etc/init/cloud-init.conf 0.7.9-243-ge74d775-0ubuntu2~
 rm_conffile /etc/init/cloud-log-shutdown.conf 0.7.9-243-ge74d775-0ubuntu2~
-rm_conffile /etc/NetworkManager/dispatcher.d/hook-network-manager 23.3-0ubuntu0~23.04.1~
+rm_conffile /etc/NetworkManager/dispatcher.d/hook-network-manager 23.3-0ubuntu1~
```

Comment:
Now that we're going to SRU a 23.1.1, I'm not sure if this should be updated to `23.3.1-0ubuntu1~` 

In the dpkg-maintscript-helper manpage:
```
    If the conffile has not been shipped for several versions, and you are
    now modifying the maintainer scripts to clean up the obsolete file,
    prior-version should be based on the version of the package that you
    are now preparing, not the first version of the package that lacked the
    conffile
```

Reply:
```suggestion
rm_conffile /etc/NetworkManager/dispatcher.d/hook-network-manager 23.3.1-0ubuntu1~
```

Reply:
Agreed. we know we are SRUing 23.3.1 not 23.3 so let's plan for that